decoder/vaapidecoder_vp9.cpp

```diff
@@ -90,18 +90,26 @@ void VaapiDecoderVP9::flush(void)
 Decode_Status VaapiDecoderVP9::ensureContext(const Vp9FrameHdr* hdr)
 {
     // only reset va context when there is a larger frame
-    if (m_configBuffer.width < hdr->width
```

Comment:
why we need change so many code, can we just call VaapiDecoderBase::start in vp9 decoder's start?